### PR TITLE
update changelog.md

### LINKED ISSUE #387: Release Notes for RTM.
Opening this to track changes for RC2.

OpenIdConnectConfiguration, JsonWebKey, JsonWebKeySet now return collections that users will need to add / remove items rather than setting the entire collection.

### PR DESCRIPTION
**Pull request by tushargupta51**
Fixes #387

### CODE REVIEW

**Comment by tushargupta51** [comment]
@brentschmaltz @polita

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,112 +1,24 @@
...
+* **Cryptography:** Added support for Elliptical Curve (ECDsa) algorithm.
 
-* **NOTE:** We have not added support for WsFederation and Saml in this version yet.
+## Major Changes
+* Dropped support for WsFederation and Saml.
+* `SecurityKey` is used for token validation instead of `SecurityToken`. 
+* `CryptoProviderFactory` provides all the extensibility to control and customize crypto support.
+* Dropped support for reading and writing JWT tokens embedded in XML.
+* Removed dependency on `System.IdentityModel.dll`.
+* Replaced `JavaScriptSerializer` by Json.Net serializer. Deserializing `int` results in an `Int64` object by default instead of `Int32`.
```
Comment (polita) [inline CHANGELOG.md]: Should this read "Replaced JavaScriptSerializer *with* Json.NET serializer"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,112 +1,24 @@
...
+* Replaced the static ClaimType maps on `JwtSecurityTokenHandler` by instances.
```
Comment (polita) [inline CHANGELOG.md]: Should "by" read "with"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,112 +1,24 @@
...
 ## Features
 * **JsonWebKey:** Upgraded to fully support features in http://tools.ietf.org/html/draft-ietf-jose-json-web-key-37
 * **Logging:** Added Event Source based logging in Wilson. `IdentityModelEventSource` is the exposed event source handle.
-* The OM removed support for `SecurityTokens` for crypto operations
-
-## Breaking Changes
+* **Cryptography:** Added support for Elliptical Curve (ECDsa) algorithm.
 
-* **NOTE:** We have not added support for WsFederation and Saml in this version yet.
+## Major Changes
```
Comment (polita) [inline CHANGELOG.md]: Perhaps this should say "Major Changes From ASP.NET 4" or whatever Katana's official name was.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,112 +1,24 @@
...
 * **Refactoring**
-    * `Microsoft.IdentityModel.Protocol.Extensions` is refactored into:
-        * **Microsoft.IdentityModel.Protocol**: Includes protocol agnostic classes, e.g. `ConfigurationManager`.
-        * **Microsoft.IdentityModel.Protocols.OpenIdConnect**: Includes OpenIdConnect specific support like `OpenIdConnectMessage`, `OpenIdConnectConfiguration`, `OpenIdConnectProtocolValidation` etc.
-        * **Microsoft.IdentityModel.Protocols.WsFederation**: Includes stubs for `WsFederation` support.
-    * `System.IdentityModel.Tokens` is refactored into:
-        * **System.IdentityModel.Tokens**: Includes support for crypto operations and other classes that are not token format specific e.g. `TokenValidationParameters`.
+    * Microsoft.IdentityModel.Protocol.Extensions is refactored into:
+        * **Microsoft.IdentityModel.Protocols**: Includes protocol agnostic classes, e.g. `ConfigurationManager`.
+        * **Microsoft.IdentityModel.Protocols.OpenIdConnect**: Includes OpenIdConnect specific support like `OpenIdConnectMessage`, `OpenIdConnectConfiguration`, `OpenIdConnectProtocolValidator` etc.
```
Comment (polita) [inline CHANGELOG.md]: Replace "like" with "for"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,112 +1,24 @@
 <a name="5.0.0"></a>
 # 5.0.0 (2015-06-01)
```
Comment (polita) [inline CHANGELOG.md]: Update the date?

**Comment by polita** [comment]
After small changes, :shipit:

File: CHANGELOG.md (hunk 4 above)
Comment (tushargupta51) [inline CHANGELOG.md]: Not sure if "for" is right. I will replace that with "e.g.".

File: CHANGELOG.md (hunk 1 above)
Comment (tushargupta51) [inline CHANGELOG.md]: "by" should be replaced _by_ "with" here. Source: http://english.stackexchange.com/a/151352 :smile:

File: CHANGELOG.md (hunk 3 above)
Comment (tushargupta51) [inline CHANGELOG.md]: I don't think we should mention Katana or asp.net here. Should just say. Major Changes from 4.x? @brentschmaltz ?

**Pull request merged**